public/components/application-ui-headers/3.html

```diff
@@ -0,0 +1,24 @@
+<header class="p-12">
```

Comment:
Should really have a container, rather than just `p-12`. Other headers will have a container you can use.